src/Serializer/tests/Serializer/ProtoSerializerTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Spiral\Tests\Serializer\Serializer;
+
+use Google\Protobuf\Internal\Message;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\TestCase;
+use Spiral\Serializer\Exception\InvalidArgumentException;
+use Spiral\Serializer\Serializer\ProtoSerializer;
+
+final class ProtoSerializerTest extends TestCase
```

Comment:
Try to get real message class for tests and use it in tests.
 https://github.com/spiral/roadrunner-bridge/tree/3.x/tests/generated/Service

Reply:
Added tests with real message.